ci-operator/step-registry/redhat-chaos/pod-scenarios/kube-apiserver-disruption/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
```

Comment:
lets use a symbolic link to the owners file in the base redhat-chaos folder like: https://github.com/openshift/release/blob/master/ci-operator/step-registry/redhat-chaos/pod-scenarios/etcd-disruption/OWNERS

Reply:
Done, thanks.